### PR TITLE
fixing overflow with links in contact page

### PR DESCRIPTION
**Pull request by KingMedieval**
fixes an issue where contact page gets cut off on some devices. It also adds a bottom border to ResultScrollView.tsx that scales to the device, so it doesn't look like the component is being cut off of the screen. Sulphine's LinkedIn link is same as Michael's. Long links also break, so it doesn't overflow on phones. Links starting with www links internally so added https.

### CODE REVIEW

**Comment by chalkedgoose** [comment]
@KingMedieval Thank you for your contribution the project you are the first external contribution so 
we haven't bothered to list any requirements for a PR. 

In the future would you mind 
1. Making a more detailed PR title ("UI Fixes" may be hard to grep later down the line)
2. Linking an issue from the project issues panel
3. Including before and after photos of the described changes

File: src/pages/contact.tsx [hunk 1]
```diff
@@ -8,38 +8,38 @@ import {
...
+            <strong>Michael Lavery</strong>
+            {" supplied the web-scraped grade and rating data. "}
+            <strong>Sulphine Susie</strong>
+            {" designed the logo and inspired the user interface. "}
+          </ContainerText>
+          <ContainerText title={"Charles O'Hanlon"}>
+            {"Github: "}
+            <ContainerLink href="https://github.com/charlescohanlon"></ContainerLink>
+            {"\nLinkedin: "}
+            <ContainerLink href="https://linkedin.com/in/charlescohanlon"></ContainerLink>
```
Comment (charlescohanlon) [inline src/pages/contact.tsx:27]: Thank you, but I've already fixed the broken links.

File: src/frontend/components/ResultScrollView.tsx [hunk 2]
```diff
@@ -3,20 +3,21 @@ interface ResultScrollContainerProps {
 }
 
 const ResultScrollView = ({
-  children,
-}: ResultScrollContainerProps): JSX.Element => {
+                            children,
+                          }: ResultScrollContainerProps): JSX.Element => {
```
Comment (charlescohanlon) [inline src/frontend/components/ResultScrollView.tsx:7]: The ResultScrollContainer cutting off at the bottom was a stylistic choice.

File: src/frontend/components/DisplayContainer/ContainerLink.tsx [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-      </a>
-    </Link>
-  );
+                                  href,
+                                  children,
+                              }: ContainerLinkProps): JSX.Element => {
+    return (
+        <Link href={href} target="_blank" rel="noopener noreferrer">
+            <a
+                className={"underline text-blue-600 decoration-blue-600 decoration-2 break-all"}
```
Comment (charlescohanlon) [inline src/frontend/components/DisplayContainer/ContainerLink.tsx:14]: Would you mind refactoring your PR just to include this change or opening a new one? This is a great suggestion, and I would love to work with you to get it on the codebase.

**Review by charlescohanlon — CHANGES_REQUESTED** [review]

**Comment by KingMedieval** [comment]
the latest commit addresses the refactoring you mentioned. Sorry, I'm new to contributing and I am trying to learn github other than using it to host code.

**Review by charlescohanlon — APPROVED** [review]
lgtm!

**Pull request merged**